packages/mini-browser/src/browser/mini-browser-content.ts

```diff
@@ -395,33 +400,95 @@ export class MiniBrowserContent extends BaseWidget {
         pdfContainer.id = `${this.id}-pdf-container`;
         pdfContainer.style.display = 'none';
 
+        contentArea.appendChild(errorBar);
         contentArea.appendChild(transparentOverlay);
         contentArea.appendChild(pdfContainer);
         contentArea.appendChild(loadIndicator);
         contentArea.appendChild(frame);
 
         parent.appendChild(contentArea);
-        return Object.assign(contentArea, { frame, loadIndicator, pdfContainer, transparentOverlay });
+        return Object.assign(contentArea, { frame, loadIndicator, errorBar, pdfContainer, transparentOverlay });
     }
 
     protected createIFrame(): HTMLIFrameElement {
         const frame = document.createElement('iframe');
         const sandbox = (this.props.sandbox || MiniBrowserProps.SandboxOptions.DEFAULT).map(name => MiniBrowserProps.SandboxOptions[name]);
         frame.sandbox.add(...sandbox);
         this.toDispose.push(addEventListener(frame, 'load', this.onFrameLoad.bind(this)));
+        this.toDispose.push(addEventListener(frame, 'error', this.onFrameError.bind(this)));
         return frame;
     }
 
+    protected createErrorBar(): HTMLElement & Readonly<{ message: HTMLElement }> {
+        const errorBar = document.createElement('div');
+        errorBar.classList.add(MiniBrowserContent.Styles.ERROR_BAR);
```

Comment:
Maybe `.noselect`.

<img width="375" alt="screen shot 2019-02-28 at 21 33 08" src="https://user-images.githubusercontent.com/1405703/53596697-9e690680-3ba0-11e9-927e-47568c6a9c58.png">

Reply:
Thanks for this suggestion, but as a user, I hate it when apps prevent me from copying error messages (e.g. what if I want to google that error message, or file a bug?)

Do you have a strong preference for using `.noselect`?

Reply:
I do not have a preference unless it is the same everywhere.

Error bar in mini-browser:
![screencast 2019-03-01 13-28-25](https://user-images.githubusercontent.com/1405703/53638330-2cd39b80-3c26-11e9-911a-3721f74912c8.gif)


Error bar in Git view:
![screencast 2019-03-01 13-28-51](https://user-images.githubusercontent.com/1405703/53638318-2513f700-3c26-11e9-8d6d-7d5b6afef477.gif)

Reply:
Oh wow, there is a similar error bar in the Git view too. Thanks, I didn't know about it!

Also it seems selectable, so I'll leave the Preview error bar selectable as well.

Reply:
> return this.doSet(context, this.memento.before);

That's the point. See the screencast. I could not select the error message from the iframe as I could from the Git view.

Reply:
Additional thought, to preserve consistency, maybe we should replace:
- `(x) Error: Loading this page is taking longer than usual` with
- `(!) Loading this page is taking longer than usual`?

Reply:
Then again maybe the more JS-error-like format works better for iframe errors.